DOCS/interface-changes.rst

```diff
@@ -27,6 +27,7 @@ Interface changes
       ("") uses the value of --hwdec.
     - drop deprecated --ad/--vd features
     - drop deprecated --sub-codepage syntax
+    - add --ontop-level option for macOS
```

Comment:
@wm4 In this case it seems reasonable to put it there, to alert the few (any?) people who want to use the "system" window level on macOS.